### PR TITLE
add labels_aave_v2_annual_liquidations data model and schema source

### PR DESCRIPTION
**Pull request by PaulApivat**
First PR raised for open competition query label submission. Marking 'x' for relevant checks below.

Brief comments on the purpose of your changes. 

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @PaulApivat 
[Sample labels](https://dune.com/queries/2209429?d=1)

Please have a final review before merging @andrewhong5297

**Comment by andrewhong5297** [comment]
looks good to me! is it possible to add more liquidators to this model? This might be a good opportunity to add a liquidations model in general to spellbook - @PaulApivat would you be up for that? I can help out as well.

File: models/labels/__single_category_labels__/aave_v2/labels_aave_v2_annual_liquidations.sql
Comment (jeff-dude) [inline models/labels/__single_category_labels__/aave_v2/labels_aave_v2_annual_liquidations.sql:28]: please use jinja syntax here for a source

**Review by jeff-dude — CHANGES_REQUESTED** [review]
after below requested change, are we good to merge and any follow-up will be separate PR?

File: models/labels/__single_category_labels__/aave_v2/labels_aave_v2_annual_liquidations.sql
Comment (PaulApivat) [reply models/labels/__single_category_labels__/aave_v2/labels_aave_v2_annual_liquidations.sql:28]: Hi @jeff-dude 

The `FROM aave_v2_annual_liquidations a` reference on line 26 refers to a previous CTE within the same .sql file on line 10 (it coincidentally has the same name as the file, if that's confusing, I can change the .sql file name itself)

In this scenario, is jinja syntax required? 

if yes, i'm thinking it would it be something like: 

`FROM {{ aave_v2_annual_liquidations }} a`   -- would this be the way to go?

Thank you.

Comment (jeff-dude) [reply models/labels/__single_category_labels__/aave_v2/labels_aave_v2_annual_liquidations.sql:28]: i'm referring to `prices.tokens` as a source, not the `aave_v2_annual_liquidations` CTE

Comment (jeff-dude) [reply models/labels/__single_category_labels__/aave_v2/labels_aave_v2_annual_liquidations.sql]: oops, meant ref, not source. `prices.tokens` is a spell, while `prices.usd` is a source. i misread that.

**Review by jeff-dude — APPROVED** [review]
all set 👍

**Comment by PaulApivat** [comment]
Thank you @jeff-dude for your review. 

Next, i'll open a new branch to raise a new PR for a query label on aave_v2 daily deposit size. cc: @andrewhong5297

**Pull request merged**
